### ISSUE TITLE
How to get underlaying vue model data on drop event handler?

### ISSUE THREAD

**Issue by caleywoods** [kristianmandrup/vue2-dragula#14]
I have pulled vue2-dragula into a demo application for [Vuetiful Datatable](https://github.com/andrewcourtice/vuetiful/) and I've gotten the basic drag and drop functionality working well (it's really quite smooth!). Here's an [example screenshot](http://i.imgur.com/afgg3yu.png) so you can see what I am working with, each of the generated `<td>` elements in there is draggable.

I have the datatable loading rows after a lodash debounced ajax call. Each of my records coming back contain many properties , one of these is `display_order`. The data I am dealing with involves invoices so we're showing line items in the datatable and this display_order property controls in what order they show.

I have setup a drop event which is wired up, it looks like this:
```
/* on my .vue file */
created () {
    let service = this.$dragula.$service

    service.eventBus.$on('drop', this.handleDrop)
},
methods : {
    handleDrop(e) {
        /* e contains the data in the screenshot below 
            I'd like to access the display_order of my row here so I can sync it with the new order of the row
            I've noticed that e.el.rowIndex shows the new index of the row but I don't have access to the model
            that I can see
        */
    }
}
```

["e" data example](http://i.imgur.com/3ztSWIB.png)

I've tried removing the `handleDrop()` method and going with the handler as a function within the `service.eventBus.$on('drop', (e) => {})` but noticed the data looking the same way. I have seen that `e.drake.models[0].model` seems to contain the datacell records but trying to fetch the model by its `rowIndex` has not worked (it's always off and shows me the wrong record).

This could be handled on the server side when I submit my rows and just update the display order based on what order they appear in the rows but I assume later on I may want to access some other properties on the models for a different purpose.

TL;DR - Does there exist a way of defining a handler for the drop event of a table row (a datacell) so that you can access the underlaying properties of your model?

**Comment by kristianmandrup** [comment]
Wow, I'm really impressed. Sounds like a very cool component that I might find useful myself.
TBH, I haven't worked much with the internals of Dragula drag'n drop and found it hard to find good, in depth documentation of the api and especially the internals behind it. I don't have any suggestions for how to best operate on the rowIndex, however you might wanna look in the `ModelManager` class and perhaps subclass it. I believe it uses the rowIndex to slice the array (by default)

**Comment by caleywoods** [comment]
@kristianmandrup Thanks, I remember at least coming across a reference to that in the normal Dragula docs, I'll go read it more closely.

**Comment by kristianmandrup** [comment]
The `ModelManager` class i internal to `vue2-dragula`, ie. used to manage the internal model. See the demo app.
